tests/errors/test_list_const.py

```diff
@@ -0,0 +1,14 @@
+from lpython import i32, list, Const
+
+def test_list_const():
+    CONST_INTEGER_LIST: Const[list[i32]] = [1, 2, 3, 4, 5, 1]
+
+    CONST_INTEGER_LIST.append(6)
+    CONST_INTEGER_LIST.insert(3, 8)
+    CONST_INTEGER_LIST.reverse()
+    CONST_INTEGER_LIST.pop()
+    CONST_INTEGER_LIST.pop(0)
+    CONST_INTEGER_LIST.clear()
+    CONST_INTEGER_LIST.remove(1)
```

Comment:
For each of these operations, there should a separate, small/consice, individual error reference test added.

Adding all in one will only trigger the very first `SemanticError()` and the rest of them are not tested/verified.

Reply:
Hmm..got it @Shaikh-Ubaid ! The separate files can be called `test_const_list_*`, right?

Reply:
>The separate files can be called test_const_list_*

Yes, you can start the numbering with `test_const_list_01.py`.